### PR TITLE
aws: add support for aws_lb/aws_alb/aws_elb resources

### PR DESCRIPTION
**Pull request by patrislav**
This PR adds support for the following resources from the `AWSELB` service:
- `aws_elb` - Classic Load Balancer
- `aws_lb`/`aws_alb` - Load Balancer v2

In order to estimate these properly, the `AWSELB` service pricing must be ingested first.

### CODE REVIEW

File: aws/terraform/provider.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ func NewProvider(key string, regionCode region.Code) (*Provider, error) {
 
 // ResourceComponents returns Component queries for a given terraform.Resource.
 func (p *Provider) ResourceComponents(tfRes terraform.Resource) []query.Component {
+	fmt.Println("Provider:", p.key, p.region, " | Resource:", tfRes.Address)
```
Comment (xescugc) [inline aws/terraform/provider.go:28]: Nop

**Comment by xescugc** [comment]
And the CI is failing

File: aws/terraform/provider.go (hunk 1 above)
Comment (patrislav) [reply aws/terraform/provider.go]: :facepalm: Fixed

**Review by xescugc — COMMENTED** [review]
rs

**Comment by patrislav** [comment]
Rebased

**Review by xescugc — APPROVED** [review]

**Pull request merged**